pvactools/tools/pvacview/anchor_and_helper_functions.R

```diff
@@ -80,6 +81,42 @@ peptide_coloring <- function(hla_allele, peptide_row) {
   colors <- colorRampPalette(c("lightblue", "blue"))(99)[value_bins]
   return(colors[[position]])
 }
+#calculate per-length anchor score for HLA allele
+anchor_weights_for_alleles <- function(hla_alleles) {
+  scores_df <- data.frame()
+  for (hla_allele in hla_alleles) {
+    if (any(hla_allele == anchor_data[[8]]["HLA"])) {
+      eight_mer_scores <- append(anchor_data[[8]][anchor_data[[8]]["HLA"] == hla_allele][1:(8 + 1)], "9mer", 1)
```

Comment:
Sorry I'm not great at reading code and immediately realizing what its doing without testing it in R, but is the "9mer" part supposed to be "8mer" here?

Reply:
Uh, doh. I did so much trial and error here to get it to work, I totally missed it. Good catch.